astronomer-starship-provider/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = astronomer-starship-provider
-version = 0.1.2
+version = 0.1.3
```

Comment:
@aspain - FYI this is what the check was looking for `0.1.2` already exists on PyPi so the deploy would fail.